training-labs/src/main/java/ai/skymind/training/solutions/LstmSingleTask.java

```diff
@@ -0,0 +1,255 @@
+package ai.skymind.training.exercises;
+import org.datavec.api.records.reader.SequenceRecordReader;
+import org.datavec.api.records.reader.RecordReader;
+import org.datavec.api.records.reader.impl.csv.CSVSequenceRecordReader;
+import org.datavec.api.records.reader.impl.transform.TransformProcessSequenceRecordReader;
+import org.datavec.api.split.NumberedFileInputSplit;
+//import org.datavec.api.records.reader.impl.transform.TransformProcessRecordReader;
+import org.datavec.api.transform.schema.Schema;
+import org.datavec.api.transform.schema.SequenceSchema;
+import org.datavec.api.transform.TransformProcess;
+import org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator;
+import org.deeplearning4j.eval.ROC;
+import org.deeplearning4j.nn.api.OptimizationAlgorithm;
+import org.deeplearning4j.nn.conf.ComputationGraphConfiguration;
+import org.deeplearning4j.nn.conf.GradientNormalization;
+import org.deeplearning4j.nn.conf.NeuralNetConfiguration;
+import org.deeplearning4j.nn.conf.BackpropType;
+import org.deeplearning4j.nn.conf.Updater;
+import org.deeplearning4j.nn.conf.inputs.InputType;
+import org.deeplearning4j.nn.conf.layers.GravesLSTM;
+import org.deeplearning4j.nn.conf.layers.LossLayer;
+import org.deeplearning4j.nn.conf.layers.RnnOutputLayer;
+import org.deeplearning4j.nn.graph.ComputationGraph;
+import org.deeplearning4j.nn.weights.WeightInit;
+import org.deeplearning4j.optimize.listeners.ScoreIterationListener;
+import org.nd4j.linalg.activations.Activation;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.dataset.api.DataSet;
+import org.nd4j.linalg.lossfunctions.impl.LossBinaryXENT;
+import org.nd4j.linalg.lossfunctions.LossFunctions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.nd4j.linalg.dataset.api.iterator.DataSetIterator;
+import org.datavec.api.transform.transform.column.RemoveColumnsTransform;
+
+/**
+ * EXERCISE 4: train a LSTM to predict mortality using the Physionet
+ * Challenge 2012 data publicly available at https://physionet.org/challenge/2012/
+ *
+ *
+ */
+public class LstmSingleTask {
```

Comment:
Rename class (and file) to PhysionetMultivariateTimeSeriesClassification.java. Assume @tomthetrainer approves that name. ;)